src/families/tezos/DelegateFlow/steps/StepConfirmation.js

```diff
@@ -85,26 +93,35 @@ function StepConfirmation({
             <IconTriangleWarning height={16} width={16} />
           </Box>
           <Box style={{ display: 'block' }} ff="Inter|SemiBold" fontSize={3} horizontal shrink>
-            <Trans i18nKey="send.steps.confirmation.broadcastError" />
+            <Trans i18nKey="delegation.flow.steps.confirmation.broadcastError" />
           </Box>
         </Disclaimer>
       ) : null}
-      <TrackPage category="Send Flow" name="Step Confirmed" />
+      <TrackPage
+        category={undelegating ? 'Undelegation Flow' : 'Delegation Flow'}
```

Comment:
Delegation Flow is the category wording we use for both. let's Delegation Flow and have that info a meta field, because otherwise it will be weird in analytics

Reply:
alright